Build/BUILDsis2

```diff
@@ -0,0 +1,98 @@
+#* useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+#* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#* See the GNU General Public License for more details.
+#*
+#* You should have received a copy of the GNU Lesser General Public
+#* License along with theSHiELD Build System
+#* If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+#  DISCLAIMER: This script is provided as-is and as such is unsupported.
+#
+set -x
+#
+# set default values
+# configure your build parameters
+  COMPILER="intel"
+  BIT="64bit"
+#
+# parse arguments
+  for arg in "$@"
+  do
+      case $arg in
+          intel|gnu)
+             COMPILER="${arg#*=}"
+             shift # Remove "compiler" from processing
+             ;;
+	  32bit|64bit)
+             BIT="${arg#*=}"
+             shift # Remove *bit from processing
+             ;;
+          *)
+          if [ ${arg#} != '--help' ] && [ ${arg#} != '-h' ] ; then
+            echo "option "${arg#}" not found"
+          fi
+          echo -e ' '
+          echo -e "valid options are:"
+          echo -e "\t[intel(D) | gnu] \t\t\t compiler"
+	  echo -e "\t[32bit(D) | 64bit] \t\t\t FV3 precision option"
+          echo -e "\n"
+          exit
+          ;;
+      esac
+  done
+
+# set up some default variables if not called from COMPILE
+# BUILD_ROOT is set if this script is called from the COMPILE script
+if [ -z ${BUILD_ROOT} ] ; then
+   export BUILD_ROOT=${PWD%/*}
+   export SHiELD_SRC=${PWD%/*/*}/SHiELD_SRC/
+   export PATH="${BUILD_ROOT}/mkmf/bin:${BUILD_ROOT}/Build/mk_scripts:${PATH}"
+   export NCEP_DIR=${BUILD_ROOT}/Build
+   if [ ! -z ${EXTERNAL_LIBS} ] ; then
+      export NCEP_DIR=${EXTERNAL_LIBS}
+   fi
+   # load the proper environment for your machine
+   . ${BUILD_ROOT}/site/environment.${COMPILER}.sh
+fi
+
+
+mkdir -p ${BUILD_ROOT}/Build/sis2
+list_paths -l -o ${BUILD_ROOT}/Build/sis2/pathnames_sis2 \
+    ${SHiELD_SRC}/mom6/src/SIS2/config_src/dynamic_symmetric \
+    ${SHiELD_SRC}/mom6/src/SIS2/config_src/external/Icepack_interfaces \
+    ${SHiELD_SRC}/mom6/src/SIS2/src \
+    ${SHiELD_SRC}/mom6/src/icebergs/src \
+    ${SHiELD_SRC}/ice_param 
+
+cd ${BUILD_ROOT}/Build
+
+pushd sis2
+
+mkmf -m Makefile -a ${SHiELD_SRC} -p libsis2.a -t "${BUILD_ROOT}/${TEMPLATE}" -c "-DINTERNAL_FILE_NML -g -DUSE_FMS2_IO -I${NCEP_DIR}/mom6 -I${NCEP_DIR}/libFMS/${COMPILER}/${BIT}" -I${SHiELD_SRC}/FMS/axis_utils/include -I${SHiELD_SRC}/FMS/diag_manager/include -I${SHiELD_SRC}/FMS/fms/include -I${SHiELD_SRC}/FMS/fms2_io/include -I${SHiELD_SRC}/FMS/horiz_interp/include -I${SHiELD_SRC}/FMS/include -I${SHiELD_SRC}/FMS/mpp/include -I${SHiELD_SRC}/FMS/sat_vapor_pres/include -I${SHiELD_SRC}/FMS/string_utils/include -I${SHiELD_SRC}/FMS/test_fms/fms/include -I${SHiELD_SRC}/mom6/src/MOM6/pkg/CVMix-src/include -I${SHiELD_SRC}/mom6/src/MOM6/src/framework ${BUILD_ROOT}/Build/sis2/pathnames_sis2
```

Comment:
I think we should break this up into multiple lines using a `\` at the end of each line

Reply:
done